### PR TITLE
LPS-129874 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
Tests passed here: https://github.com/lawrence-lee/liferay-portal/pull/207#issuecomment-811587468

File: modules/build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ ext {
 		"com.liferay.gradle.plugins.css.builder.version": "5.0.1",
 		"com.liferay.gradle.plugins.rest.builder.version": "1.1.21",
```
Comment (gamerson) [inline modules/build.gradle]: Do we still need this token anymore? `"com.liferay.gradle.plugins.rest.builder.version": "1.1.21",`

Comment (lawrence-lee) [reply modules/build.gradle]: Removed

**Comment by petershin** [comment]
ci:forward

**Comment by lawrence-lee** [comment]
ci:forward

**Pull request closed without merging**